pilot/pkg/xds/util.go

```diff
@@ -41,3 +45,39 @@ func getSubSetLabels(dr *networkingapi.DestinationRule, subsetName string) label
 
 	return nil
 }
+
+func IsThisVersionOlderThanThat(this, that string) (bool, error) {
+	//format is "time.Now().Format(time.RFC3339) + "/" + strconv.FormatUint(versionNum.Inc(), 10)"
```

Comment:
Version is an opaque string, I don't think we should be parsing it. It is fragile and slow